client/src/legacy/LeftAndMain.Panel.js

```diff
@@ -157,6 +179,18 @@ $.entwine('ss', function($) {
 
       this.trigger('toggle', doExpand);
       this.trigger(doExpand ? 'expand' : 'collapse');
+
+      // Update decendant button
+      const button = this.find('.cms-panel-toggle__button');
+      if (doExpand) {
+        button.attr('aria-expanded', 'true');
+        button.attr('title', button.attr('data-expanded-title'));
+        button.html(button.attr('data-expanded-label'));
```

Comment:
Need spans with `aria-hidden="true"` as per https://github.com/silverstripe/silverstripe-admin/pull/1978#pullrequestreview-3153168151

The alternative is for this change in the templates:
```diff
- data-expanded-label="&laquo;"
+ data-expanded-label="<span aria-hidden="true">&laquo;</span>"
```

Reply:
Added aria-label instead e.g. "Collapse Panel" or "Expand Panel"